### PR TITLE
Working function of adding even numbers

### PR DESCRIPTION
**Pull request by opiotrowsky**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @opiotrowsky

**Comment by ziobron** [comment]
⏱ 3 XP extra for @opiotrowsky for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,15 @@
...
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    std::vector<int> evens;
+    int suma = 0;
+    for(const size_t& el : numbers) {
+        if(el % 2 == 0) {
+            evens.push_back(el);
+        }
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:10]: you can sum here

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
+    }
+    for(const size_t& el : evens) {
+        suma += el;
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:13]: redundant

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**